sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/CreateNotification.groovy

```diff
@@ -0,0 +1,29 @@
+package life.qbic.business.notification.create
+
+import life.qbic.business.subscription.Subscriber
+
+import java.time.LocalDate
+
+/**
+ * <h1>Sends an email to a lists of subscribers</h1>
+ *
+ * <p>This use case collects a list of subscribers who's subscribed project got updated the current day</p>
+ *
+ * @since 1.0.0
+ *
+*/
+class CreateNotification implements CreateNotificationInput{
+    private final CreateNotificationDataSource ds
+
+    CreateNotification(CreateNotificationDataSource ds, CreateNotificationOutput output){
+        this.ds = ds
+    }
+
+    @Override
+    void createNotifications() {
+        //todo when do we want to run the tool? Is "today" sufficient or do we want to do it for the day before?
+        List<Subscriber> subscribers = ds.getSubscribersForTodaysNotifications(LocalDate.now().minusDays(1))
```

Comment:
I would move the logic of getting all notifications and then all subscribers for all the projects in here instead of the database.

Reply:
I was already also thinking about that, will do